docs/distributions/endeavouros/installation.md

```diff
@@ -0,0 +1,63 @@
+# Installing EndeavourOS on a Mac with the T2 Chip
+
+You will need:
+
+- USB drive with at least 4GB
+- A way to plug it into your Mac (USB-C isn't USB-A)
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi.
+
+---
+
+1. Making a partition for Linux.
+
+    1. Open the Bootcamp installer and follow it until it asks for a Windows ISO, this will clear space for a Linux partition (by removing APFS snapshots).
+    2. In macOS Disk Utility, make a partition, format doesn't matter, but pick the amount of space that you want for Linux. You won't be able to resize your APFS partitions from the installer, so you must make space now.
+
+2. Creating bootable media
+
+    1. Download an installer ISO from [here](https://github.com/t2linux/EndeavourOS-ISO-t2/releases).
+    2. Put this image onto a USB stick, follow these instructions on the [Arch Wiki](https://wiki.archlinux.org/index.php/USB_flash_installation_medium#In_macOS).
+
+3. Disabling secure boot
+
+    1. Follow [this article's](https://support.apple.com/en-us/HT208198) instructions.
+    2. Once in startup security utility, turn secure boot to no security and enable external boot.
+
+4. Booting the live environment.
+
+    1. Plug the USB in to your computer.
+    2. Boot while holding the option key, this will put you in macOS Startup Manager.
+    3. Select the orange EFI option with arrow keys and press return/enter on it.
+
+5. Starting Calamares installer
+
+    1. If you're not connected to the internet, connect to it now. Use included GUI config tool to connect to Wi-Fi.
+    2. Click "Start the Installer" on the "Welcome" window.
+
+        !!! warning
+            NEVER use the option "Install community editions" or choose "Offline" installation method. They certainly don't work.
```

Comment:
maybe suggest where to find instructions for installing other desktop environments after doing the offline install

Reply:
Since the installer borks out when installing offline, I think it's not worth the effort. And it's unsupported.